Directory.Packages.props

```diff
@@ -0,0 +1,44 @@
+<Project>
```

Comment:
Could you expand the PR description to explain that we are implementing central package management to pin transient dependencies to secure versions as well as to reduce the duplication of version information and the risk of discrepancies? https://cbea.ms/git-commit/#why-not-how

Also, could you change the title to imperative mood? https://cbea.ms/git-commit/#imperative